README.md

```diff
@@ -116,93 +116,95 @@ ninja check-npcomp
 export PYTHONPATH="$(realpath python):$(realpath build/python)"
 ```
 
-### PyTorch 1.3 - ATen pseudo-device type dispatch
+### PyTorch Frontend Build (manual docker instructions)
 
-The currently functional approach to PyTorch integration uses an ATen pseudo
-device for program capture. It is activated by including the PyTorch cmake
-path and settind `-DNPCOMP_ENABLE_TORCH_TYPE_DISPATCH=ON`. This approach has a
-very fragile dependency on a specific PyTorch revisions in the ~1.3 era and
-currently must be built via the docker image in `docker/pytorch-1.3`.
+Create docker image (or follow your own preferences):
 
-We are migrating to newer approaches that build with more recent PyTorch
-versions, but these are not yet functional (see below).
+* Map the source directory to `src/mlir-npcomp`
+* Map the `/build` directory appropriately for your case.
 
-Docker container setup:
+```shell
+docker build docker/pytorch-1.6 --tag local/npcomp:build-pytorch-1.6
+docker volume create npcomp-build
+```
+
+Shell into docker image:
 
 ```shell
-# One of the maintainers does periodically push new images. To use one of these,
-# skip the build step and use:
-#   BUILD_IMAGE_TAG="stellaraccident/npcomp:build-pytorch-1.3"
-# Since we are not planning to support this branch long term, this process is
-# entirely ad-hoc at present and geared for project maintainers and build bots
-# to be able to make progress.
-# See https://hub.docker.com/repository/docker/stellaraccident/npcomp
-BUILD_IMAGE_TAG="local/npcomp:build-pytorch-1.3"
-
-# Build the docker image (rebuilds PyTorch, so takes quite some time).
-docker build docker/pytorch-1.3 --tag $BUILD_IMAGE_TAG
-
-# Docker workflow (or use your own preferences).
-# Create a volume for npcomp build artifacts.
-docker volume create npcomp-pytorch-1.3-build
-
-# Run the container, mounting /npcomp to the source directory and the volume
-# above to the /build directory. The source directory is mounted read-only to
-# avoid the container putting root owned files there.
-# Replace `$HOME/src/mlir-npcomp` with an appropriate path to where the project
-# is checked out.
 docker run \
-  --mount type=bind,source=$HOME/src/mlir-npcomp,target=/npcomp,readonly \
-  --mount source=npcomp-pytorch-1.3-build,target=/build \
-  --rm -it $BUILD_IMAGE_TAG /bin/bash
+  --mount type=bind,source=$HOME/src/mlir-npcomp,target=/src/mlir-npcomp \
+  --mount source=npcomp-build,target=/build \
+  --rm -it local/npcomp:build-pytorch-1.6 /bin/bash
```

Comment:
This is so canonical, that I think the least risky thing is to just keep it to the forms that are used everywhere. The meaning is somewhat non-intuitive, but you just have to learn it. `--rm` means to delete the container on exit and `-it` means "interactive" and ensures that a console is attached (otherwise, it just pipes). Both are used almost all the time for interactive use (versus more systematic use which differs).